src/components/notifications/notifications.tsx

```diff
@@ -37,9 +37,22 @@ interface ToastPropsWithType extends ToastProps {
 }
 
 const CustomToast = ({ message, type, url }: ToastPropsWithType) => {
+  const getIcon = () => {
+    switch (type) {
+      case 'error':
+        return images.error;
+      case 'info':
+        return images.info;
+      case 'success':
+        return images.success;
+      case 'warning':
+        return images.warning;
+    }
+  };
+
   return (
     <div className={classNames(styles.notificationBody, { [styles.url]: url })}>
-      <img src={`/${type}.svg`} alt={`${type} icon`} />
+      <img src={getIcon()} alt={`${type} icon`} />
```

Comment:
Isn't `images[type]` enough?

Reply:
It would be an improvement yes 👍, but similarly to `/${type}.svg`, it doesn't show up when searching for usages of the warning, error, info, or success images. Being able to search for usages will help if/when we convert those images to inline SVGs.

Reply:
Makes sense, feel free to leave is as.